source/_components/cover.rpi_gpio.markdown

```diff
@@ -27,6 +27,8 @@ To enable Raspberry Pi Covers in your installation, add the following to your `c
 # Example configuration.yaml entry
 cover:
   platform: rpi_gpio
+  relay_time: 0.2
+  state_pull_mode: 'UP'
```

Comment:
By adding an [example](https://home-assistant.io/components/cover.command_line/#examples) section. It happens a couple of times in the past that people copied the sample configuration without using their brain and then setup failed because the values didn't match their environment. 

I guess that the "Configuration variables:" listing should be updated as well to show the right position of the entries.